### DISCUSSION TITLE
All About August ❇️ [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#63401] [New to GitHub]
Our category’s answers and contributors keep getting better and better. Thank you to all the fine folks who help those just starting on their GitHub journey. Some folks in particular the past two weeks have really crushed it…

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @Imen-Frigui: For an [answer](https://github.com/orgs/community/discussions/62392#discussioncomment-6606306) that came with a docs link and desire to learn more about the member’s question. Well done! 
* @h7x4: For a robust and kind [answer](https://github.com/orgs/community/discussions/62445#discussioncomment-6616183) that also included a docs link along with additional resources to help the asker thoroughly understand more about forking. Additional kudos for continuing the conversation with the member until they solved ✅. 
* @filefqa: Self-solver! We love to see the continued troubleshooting and coming back to your discussion with an [answer](https://github.com/orgs/community/discussions/62536#discussioncomment-6652469). Pat yourself on the back! 
 
 💫 Want to nominate a community member for a feature above? Just tag me         (@queenofcorgis) where you’ve seen great community behavior. 💫

🆕 **What’s new?** The latest news and releases from GitHub and the community. 
* **Crafting an Accessible and Inclusive Resumé:** @streats put together a comprehensive [guide in her Discussion post](https://github.com/orgs/community/discussions/62531) on making your resume simpler, easier to read, and more compatible with assistive technology 🙇. 
* **Using GitHub Apps**: Looking to extend the functionality of GitHub? @skedwards88 has a [discussion post](https://github.com/orgs/community/discussions/61290) with an overview on how to build and use GitHub apps. 

📚 **Rousing Reads and Beyond**: GitHub-produced goodies 🍭
* **A checklist and guide to get your repository collaboration-ready**: In the world of software development, collaboration can make the difference between a brittle last-minute release and a reliable, maintainable, pain-free project, @peckjon share settings, content, and behavior to make your collaboration a success in his [blogpost](https://github.blog/2023-08-04-a-checklist-and-guide-to-get-your-repository-collaboration-ready/).  
* **Make your first open source contribution in four easy steps**: A beginner’s [guide](https://github.com/readme/guides/first-oss-contribution) to make an impact with open source contributions from @Ruth-Ikegah. 
* **From chaos to clarity**: Learn about code visibility to illuminate unfamiliar code in this [guide](https://github.com/readme/guides/code-visibility-codesee) from the ReadME Project . 

What was your first open source contribution? Share it!

**Comment by Imen-Frigui** [comment]
> Our category’s answers and contributors keep getting better and better. Thank you to all the fine folks who help those just starting on their GitHub journey. Some folks in particular the past two weeks have really crushed it…
> 
> 🌟**Community Superstars**: A round of applause for these community citizens 👏
> 
>     * @Imen-Frigui: For an [answer](https://github.com/orgs/community/discussions/62392#discussioncomment-6606306) that came with a docs link and desire to learn more about the member’s question. Well done!
> 
>     * @h7x4: For a robust and kind [answer](https://github.com/orgs/community/discussions/62445#discussioncomment-6616183) that also included a docs link along with additional resources to help the asker thoroughly understand more about forking. Additional kudos for continuing the conversation with the member until they solved ✅.
> 
>     * @filefqa: Self-solver! We love to see the continued troubleshooting and coming back to your discussion with an [answer](https://github.com/orgs/community/discussions/62536#discussioncomment-6652469). Pat yourself on the back!
> 
> 
> 💫 Want to nominate a community member for a feature above? Just tag me (@queenofcorgis) where you’ve seen great community behavior. 💫
> 
> 🆕 **What’s new?** The latest news and releases from GitHub and the community.
> 
>     * **Crafting an Accessible and Inclusive Resumé:** @streats put together a comprehensive [guide in her Discussion post](https://github.com/orgs/community/discussions/62531) on making your resume simpler, easier to read, and more compatible with assistive technology 🙇.
> 
>     * **Using GitHub Apps**: Looking to extend the functionality of GitHub? @skedwards88 has a [discussion post](https://github.com/orgs/community/discussions/61290) with an overview on how to build and use GitHub apps.
> 
> 
> 📚 **Rousing Reads and Beyond**: GitHub-produced goodies 🍭
> 
>     * **A checklist and guide to get your repository collaboration-ready**: In the world of software development, collaboration can make the difference between a brittle last-minute release and a reliable, maintainable, pain-free project, @peckjon share settings, content, and behavior to make your collaboration a success in his [blogpost](https://github.blog/2023-08-04-a-checklist-and-guide-to-get-your-repository-collaboration-ready/).
> 
>     * **Make your first open source contribution in four easy steps**: A beginner’s [guide](https://github.com/readme/guides/first-oss-contribution) to make an impact with open source contributions from @Ruth-ikegah.
> 
>     * **From chaos to clarity**: Learn about code visibility to illuminate unfamiliar code in this [guide](https://github.com/readme/guides/code-visibility-codesee) from the ReadME Project .
> 
> 
> What was your first open source contribution? Share it!

What a pleasure

**Reply by queenofcorgis** [reply]
Thanks for being a member of the community and part of the New to GitHub category @Imen-Frigui 💟

**Comment by queenofcorgis** [comment]
[Read our latest check-in](https://github.com/orgs/community/discussions/64873) to celebrate more community members and learn what's new!
